### PR TITLE
Update client

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Review by Thoms-C — COMMENTED** [review]
console.log =>line 21 => UpdateCLient Page

**Review by Thoms-C — DISMISSED** [review]
Noice !

File: src/components/ConnexionLogin.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post("http://localhost:3001/auth/login", values)
+      .then((res) => {
+        setToken(res.data);
+        history.push("/clientpage");
+      })
+      .catch((e) => {
+        console.warn(e);
```
Comment (LoicBrassart) [inline src/components/ConnexionLogin.jsx]: No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No. No.

File: src/components/InscriptionSignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    setErrors(validateInfo(values));
+    delete values.password2;
+    axios
+      .post("http://localhost:3001/auth/signup", values)
+      .then((res) => {
+        setValues(res.data);
+        history.push("/clientpage");
+      })
+      .catch((e) => {
+        console.warn(e);
```
Comment (LoicBrassart) [inline src/components/InscriptionSignUp.jsx]: Nein ? Njet ?

File: src/components/UpdateClient.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    delete values.password2;
+    axios
+      .patch(`http://localhost:3001/client/9`, values)
+      .then((res) => {
+        setValues(res.data);
+      })
+      .catch((e) => {
+        console.warn(e);
```
Comment (LoicBrassart) [inline src/components/UpdateClient.jsx]: Yada !? Yamete kudasai !!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
You know I' ll never knowingly let pass a PR containing a `console.*`, right ? So, why do you keep writing them ?

**Review by Thoms-C — APPROVED** [review]
console.warn() delete ok !

**Review by Thoms-C — APPROVED** [review]
Console.warn() deleted !!!

File: src/components/ProfilPage.jsx [hunk 4]
```diff
@@ -15,14 +15,15 @@ import {
...
+import UpdateClient from "./UpdateClientForm.jsx";
 
 export default () => {
   const [clients, setClients] = useState([]);
   const [loading, setLoading] = useState(false);
 
   useEffect(() => {
     setLoading(true);
-    axios.get("http://localhost:3001/client/1").then((res) => {
+    axios.get("http://localhost:3001/client/9").then((res) => {
```
Comment (LoicBrassart) [inline src/components/ProfilPage.jsx:26]: This line seems random, it's typically the type of line which will break everything

**Review by LoicBrassart — APPROVED** [review]
No more major Evils

**Pull request merged**